cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py

```diff
@@ -108,37 +146,73 @@ def validate_network_name_in_port_lists(self, src_ports, dst_ports):
                         msg="Network ports is not the same. SRC: %s \n DST: %s"
                             % (src_net_names, dst_net_names))
 
-    def test_migrate_keystone_users(self):
-        """Validate users were migrated with correct name and email."""
+    def get_nosetest_cmd_attribute_val(self, attribute):
+        env = os.environ
+        manager = DefaultPluginManager()
+        if env.get('NOSE_IGNORE_CONFIG_FILES', False):
```

Comment:
Why does the test need to care about environment options? In case this is really necessary, it needs to be documented at least